examples/wallet/app/containers/CreateUnlockWalletPassword.js

```diff
@@ -0,0 +1,23 @@
+// @flow
+import { connect } from 'react-redux';
+import CreateUnlockWalletPassword from '../components/CreateUnlockWalletPassword';
+import { SET_VALID_UNLOCK_WALLET_PASSWORD } from '../actions/account';
+
+function mapDispatchToProps(dispatch) {
+  return {
+    setValidUnlockWalletPassword: (
+      unlockWalletPassword: string,
+      isValidUnlockPassword: boolean
+    ) =>
+      dispatch({
```

Comment:
It's generally a better idea to expose a function in the action file that generates the action object itself

Reply:
This function is not used anymore